README.md

```diff
@@ -1,18 +1,26 @@
-![](https://img.shields.io/badge/Microverse-blueviolet)
 
-# Webpack Boiler plate
+# Leaderboard
 
-> this is a boiler plate for starting a project that I will configure webpack in it.
->  When working on the project you will not want to run the build command from the terminal every time you make a change in the code. Therefore I will install a webpack dev server, which will watch my source files, generate compiled distribution files and even refresh the browser every time I save changes in the source code.
+> I will set up a JavaScript project for the Leaderboard list app, using webpack and ES6 features, notably modules.
+
+![image](https://user-images.githubusercontent.com/51750930/137402206-2f58e310-2db3-4afb-8530-02770885fda1.png)
+ 
 
 ## Getting Started
 
-**This is an example of how you may give instructions on setting up your project locally.**
-**Modify this file to match your project, remove sections that don't apply. For example: delete the testing section if the currect project doesn't require testing.**
+get a local copy up and running follow these simple example steps.
+open your terminal and clone the project using 
+
+`git clone https://github.com/shadabwahidullah/Leaderboard.git`
+
+install or update your npm using
+
+`npm update`
+
+start your local server and build project using
 
+`npm start`
 
-To get a local copy up and running follow these simple example steps.
-clone the project and you can start your own project on it.
 
 ## Authors
```

Comment:
- [ ] `Authors` is pluralized, yet there is one author. You can fix this.

Reply:
ok, thanks. But you had added all the comments as optional. 
I am wondering why didn't you approve it.

Reply:
I do apologize since this requirement was missed in the previous review. However, as a re-reviewer, I can only make it optional. I strongly advise you to implement this change before moving to the next project.

Reply:
Oh, I see.